modules/ROOT/pages/runtime-installation-task.adoc

```diff
@@ -84,28 +84,20 @@ $ $MULE_HOME/bin/mule install
 
 == Configure Properties in the wrapper.conf File
 
-After installing Mule as a Windows service, you can override the properties defined in the `wrapper.conf` file by declaring new properties in the `wrapper.additional.conf` file.
+After installing Mule as a service or as a daemon, you can modify the properties defined in the `wrapper.conf` file.
 
-The configuration steps change depending on the Mule edition you are using:
+For details on what you can do with the configuration file, see https://wrapper.tanukisoftware.com/doc/english/properties.html.
 
-* Mule Community Edition
-+
-. After adding a new property, reinstall or restart the Windows service.
-. Add the property, ensuring that its index is higher than the highest index in the `wrapper-additional.conf` file so that your property cannot be overridden.
-+
-For example, imagine that `wrapper-additional.conf` has the following property:
-+
-`wrapper.java.additional.n=-Dproperty=value`
-+
-If `n` is the highest index, you would add your property in the `wrapper.conf` file and increment the index, as follows:
-+
-`wrapper.java.additional.n+1=-DyourProperty=valueOfYourProperty`
+After modifying the `wrapper.conf` file, restart your Mule instance to enable the changes.
 
-* Mule Enterprise Edition
-+
-After you install Mule as a service, configure up to 20 additional properties using the following command-line argument:
-+
-`-additionalJavaProperties=numberOfExtraProperties`
+[NOTE]
+====
+When you add new properties, ensure that no other properties already use the same sequence number.
+
+Don't modify the `wrapper-additional.conf` file, as Mule automatically generates this file.
+
+In order to avoid the sequences of the properties in `wrapper-additional.conf` from colliding with they ones you added, you can stop and start Mule (instead of restart), that way the `wrapper-additional.conf` is regenerated taking your new properties into account. Another option is to use `-additionalJavaProperties=numberOfExtraProperties` to have a gap between the last sequence in `wrapper.conf` and the first one in `wrapper-additional.conf` (Mule Enterprise Edition only).
```

Comment:
```suggestion
To prevent property sequences in `wrapper-additional.conf` from colliding with the ones you added, stop and start Mule instead of restarting it. This ensures that `wrapper-additional.conf` regenerates, incorporating your new properties. Alternatively, you can use `-additionalJavaProperties=numberOfExtraProperties` to have a gap between the last sequence in `wrapper.conf` and the first one in `wrapper-additional.conf` (Mule Enterprise Edition only).
```